### ISSUE TITLE
Is elastic v6 compatible with the elasticsearch 5.x

### ISSUE THREAD

**Issue by UCSBGauchos** [olivere/elastic#808]
Please use the following questions as a guideline to help me answer
your issue/question without further inquiry. Thank you.

### Which version of Elastic are you using?

[ ] elastic.v6 (for Elasticsearch 6.x)

My question:
Is the v6 client compatible with the old elasticsearch 5.x like elasticsearch 5.1.1. Becasuse we update the elasticsearch client in our services and out elasticsearch cluster in production env is still 5.1.1.

Thanks!

**Comment by olivere** [comment]
Connecting to ES 6.x with Elastic v5 might work, but it's only tested to work with Elastic v6. It is quite simple to change the import path as specified in the README or the Wiki (`gopkg.in/elastic.v5` -> `github.com/olivere/elastic@^6.0.0`).

Also notice the [breaking changes in 6.0](https://www.elastic.co/guide/en/elasticsearch/reference/6.2/breaking-changes-6.0.html).

**Comment by UCSBGauchos** [comment]
No. I mean I want to use elastic v5 to connect es 6.x. Is it compatible?

**Comment by UCSBGauchos** [comment]
No. I mean I want to use elastic v6 to connect es 5.x. Is it compatible?

**Comment by olivere** [comment]
It might be. It's not tested, and it's not guaranteed. What's the problem with changing the import path?
